src/java.rs

```diff
@@ -50,6 +50,7 @@ use bbs::{ToVariableLengthBytes, FR_COMPRESSED_SIZE, G1_COMPRESSED_SIZE};
 use std::cell::RefCell;
 
 thread_local! {
+    #[allow(clippy::missing_const_for_thread_local)]
     static LAST_ERROR: RefCell<Option<String>> = RefCell::new(None);
```

Comment:
Nit: Could we just patch the style lint?

https://rust-lang.github.io/rust-clippy/stable/#/missing_const_for_thread_local

Reply:
Same as here https://github.com/mattrglobal/ffi-bbs-signatures/pull/80#discussion_r1865036025

Reply:
👍